### PR TITLE
Localize all amount input and display

### PR DESCRIPTION
**Pull request by paullinator**
#### PR Dependencies

 none

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS device
- [ ] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202764583533713

### CODE REVIEW

File: src/plugins/gui/scenes/EnterAmountScene.js [hunk 1]
```diff
@@ -198,3 +215,28 @@ const getStyles = cacheStyles((theme: Theme) => ({
     resizeMode: 'contain'
   }
 }))
+
+// Check of string has no more than one comma or one period
+const checkAmount = (amount: string): boolean => {
+  // const dotSplit = amount.split('.')
```
Comment (Jon-edge) [inline src/plugins/gui/scenes/EnterAmountScene.js:221]: Lots of commented code here, any reason to keep around?

**Comment by Jon-edge** [comment]
Aside from minor comment, good to go

**Review by Jon-edge — CHANGES_REQUESTED** [review]
Aside from minor comment, good to go

File: src/plugins/gui/scenes/EnterAmountScene.js (hunk 1 above)
Comment (paullinator) [reply src/plugins/gui/scenes/EnterAmountScene.js]: Ah you caught me before I got a chance to rebase and remove comments. Check it out again

**Pull request merged**